Prolog...
================================================================================
Running on worker5. This worker has IP 134.60.70.136/24
Running as marina, with groups: students docker
Job ID: 24828
CUDA_VISIBLE_DEVICES: 0
Using CPU Cores 40-49
Using 64000 MiB RAM
================================================================================
docker run -i --rm --pull=always --ipc=host -e XDG_CACHE_HOME=/tmp/cache --name slurm_24828 --cpuset-cpus 40-49 --memory 64000m --gpus '"device=4"' -w /mnt/hdd/marina/workspace/2D-to-3D-Pretraining -v /mnt/hdd/marina:/mnt/hdd/marina -p 0.0.0.0:9999:9999 pytorch/pytorch:latest bash -c '{ apt-get update;  DEBIAN_FRONTEND=noninteractive apt-get install -y sudo; python3 -m pip install -r requirements.txt; groupadd --gid 1004 students; useradd --uid 1052 -s /bin/bash --gid 1004 -m -d /mnt/hdd/marina marina; nvidia-smi -L; sudo --set-home --preserve-env=PATH --user=\#1052 -- ./start.sh; }'
================================================================================
latest: Pulling from pytorch/pytorch
Digest: sha256:11691e035a3651d25a87116b4f6adc113a27a29d8f5a6a583f8569e0ee5ff897
Status: Image is up to date for pytorch/pytorch:latest
Get:1 http://archive.ubuntu.com/ubuntu jammy InRelease [270 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [128 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [127 kB]
Get:4 http://security.ubuntu.com/ubuntu jammy-security InRelease [129 kB]
Get:5 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages [17.5 MB]
Get:6 http://archive.ubuntu.com/ubuntu jammy/restricted amd64 Packages [164 kB]
Get:7 http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages [1792 kB]
Get:8 http://archive.ubuntu.com/ubuntu jammy/multiverse amd64 Packages [266 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 Packages [53.3 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1515 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [3475 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [2751 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy-backports/main amd64 Packages [81.4 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy-backports/universe amd64 Packages [33.8 kB]
Get:15 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [1226 kB]
Get:16 http://security.ubuntu.com/ubuntu jammy-security/multiverse amd64 Packages [45.2 kB]
Get:17 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [3331 kB]
Get:18 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [2458 kB]
Fetched 35.3 MB in 2s (17.7 MB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  sudo
0 upgraded, 1 newly installed, 0 to remove and 33 not upgraded.
Need to get 821 kB of archives.
After this operation, 2568 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 sudo amd64 1.9.9-1ubuntu2.4 [821 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 821 kB in 1s (1105 kB/s)
Selecting previously unselected package sudo.
(Reading database ... (Reading database ... 5%(Reading database ... 10%(Reading database ... 15%(Reading database ... 20%(Reading database ... 25%(Reading database ... 30%(Reading database ... 35%(Reading database ... 40%(Reading database ... 45%(Reading database ... 50%(Reading database ... 55%(Reading database ... 60%(Reading database ... 65%(Reading database ... 70%(Reading database ... 75%(Reading database ... 80%(Reading database ... 85%(Reading database ... 90%(Reading database ... 95%(Reading database ... 100%(Reading database ... 6643 files and directories currently installed.)
Preparing to unpack .../sudo_1.9.9-1ubuntu2.4_amd64.deb ...
Unpacking sudo (1.9.9-1ubuntu2.4) ...
Setting up sudo (1.9.9-1ubuntu2.4) ...
Processing triggers for libc-bin (2.35-0ubuntu3.6) ...
Collecting jupyterlab (from -r requirements.txt (line 1))
  Downloading jupyterlab-4.3.3-py3-none-any.whl.metadata (16 kB)
Requirement already satisfied: numpy>=1.23.5 in /opt/conda/lib/python3.10/site-packages (from -r requirements.txt (line 2)) (1.26.3)
Collecting matplotlib>=3.5.1 (from -r requirements.txt (line 3))
  Downloading matplotlib-3.9.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (11 kB)
Collecting scipy>=1.9.3 (from -r requirements.txt (line 4))
  Downloading scipy-1.14.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 60.8/60.8 kB 2.6 MB/s eta 0:00:00
Collecting scikit-image>=0.20 (from scikit-image[data]>=0.20->-r requirements.txt (line 5))
  Downloading scikit_image-0.24.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (14 kB)
Requirement already satisfied: Pillow>=2.1.0 in /opt/conda/lib/python3.10/site-packages (from -r requirements.txt (line 6)) (10.2.0)
Requirement already satisfied: torch>=2.0.1 in /opt/conda/lib/python3.10/site-packages (from -r requirements.txt (line 7)) (2.2.1)
Requirement already satisfied: torchvision in /opt/conda/lib/python3.10/site-packages (from -r requirements.txt (line 8)) (0.17.1)
Collecting torchmetrics (from -r requirements.txt (line 9))
  Downloading torchmetrics-1.6.0-py3-none-any.whl.metadata (20 kB)
Collecting tensorboard (from -r requirements.txt (line 10))
  Downloading tensorboard-2.18.0-py3-none-any.whl.metadata (1.6 kB)
Collecting torchsummary (from -r requirements.txt (line 11))
  Downloading torchsummary-1.5.1-py3-none-any.whl.metadata (296 bytes)
Collecting einops>=0.3.0 (from -r requirements.txt (line 12))
  Downloading einops-0.8.0-py3-none-any.whl.metadata (12 kB)
Collecting timm (from -r requirements.txt (line 13))
  Downloading timm-1.0.12-py3-none-any.whl.metadata (51 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 51.6/51.6 kB 2.6 MB/s eta 0:00:00
Collecting scikit-learn>=0.18 (from -r requirements.txt (line 14))
  Downloading scikit_learn-1.6.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)
Collecting pandas>=1.3.5 (from -r requirements.txt (line 15))
  Downloading pandas-2.2.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (89 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 89.9/89.9 kB 6.2 MB/s eta 0:00:00
Requirement already satisfied: tqdm>=4.62.3 in /opt/conda/lib/python3.10/site-packages (from -r requirements.txt (line 16)) (4.65.0)
Collecting transformers>=4.36.2 (from -r requirements.txt (line 17))
  Downloading transformers-4.47.0-py3-none-any.whl.metadata (43 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.5/43.5 kB 3.2 MB/s eta 0:00:00
Collecting accelerate>=0.26.0 (from -r requirements.txt (line 18))
  Downloading accelerate-1.2.0-py3-none-any.whl.metadata (19 kB)
Collecting medmnist (from -r requirements.txt (line 19))
  Downloading medmnist-3.0.2-py3-none-any.whl.metadata (14 kB)
Collecting monai (from -r requirements.txt (line 20))
  Downloading monai-1.4.0-py3-none-any.whl.metadata (11 kB)
Collecting fire (from -r requirements.txt (line 21))
  Downloading fire-0.7.0.tar.gz (87 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 87.2/87.2 kB 7.4 MB/s eta 0:00:00
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting acsconv (from -r requirements.txt (line 24))
  Downloading ACSConv-0.1.1.tar.gz (15 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting kaggle (from -r requirements.txt (line 25))
  Downloading kaggle-1.6.17.tar.gz (82 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 82.7/82.7 kB 5.3 MB/s eta 0:00:00
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting evaluate (from -r requirements.txt (line 26))
  Downloading evaluate-0.4.3-py3-none-any.whl.metadata (9.2 kB)
Collecting async-lru>=1.0.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading async_lru-2.0.4-py3-none-any.whl.metadata (4.5 kB)
Collecting httpx>=0.25.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading httpx-0.28.1-py3-none-any.whl.metadata (7.1 kB)
Collecting ipykernel>=6.5.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading ipykernel-6.29.5-py3-none-any.whl.metadata (6.3 kB)
Requirement already satisfied: jinja2>=3.0.3 in /opt/conda/lib/python3.10/site-packages (from jupyterlab->-r requirements.txt (line 1)) (3.1.3)
Collecting jupyter-core (from jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_core-5.7.2-py3-none-any.whl.metadata (3.4 kB)
Collecting jupyter-lsp>=2.0.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_lsp-2.2.5-py3-none-any.whl.metadata (1.8 kB)
Collecting jupyter-server<3,>=2.4.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_server-2.14.2-py3-none-any.whl.metadata (8.4 kB)
Collecting jupyterlab-server<3,>=2.27.1 (from jupyterlab->-r requirements.txt (line 1))
  Downloading jupyterlab_server-2.27.3-py3-none-any.whl.metadata (5.9 kB)
Collecting notebook-shim>=0.2 (from jupyterlab->-r requirements.txt (line 1))
  Downloading notebook_shim-0.2.4-py3-none-any.whl.metadata (4.0 kB)
Requirement already satisfied: packaging in /opt/conda/lib/python3.10/site-packages (from jupyterlab->-r requirements.txt (line 1)) (23.1)
Requirement already satisfied: setuptools>=40.8.0 in /opt/conda/lib/python3.10/site-packages (from jupyterlab->-r requirements.txt (line 1)) (68.2.2)
Requirement already satisfied: tomli>=1.2.2 in /opt/conda/lib/python3.10/site-packages (from jupyterlab->-r requirements.txt (line 1)) (2.0.1)
Collecting tornado>=6.2.0 (from jupyterlab->-r requirements.txt (line 1))
  Downloading tornado-6.4.2-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.5 kB)
Requirement already satisfied: traitlets in /opt/conda/lib/python3.10/site-packages (from jupyterlab->-r requirements.txt (line 1)) (5.7.1)
Collecting contourpy>=1.0.1 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading contourpy-1.3.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.4 kB)
Collecting cycler>=0.10 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading cycler-0.12.1-py3-none-any.whl.metadata (3.8 kB)
Collecting fonttools>=4.22.0 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading fonttools-4.55.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (165 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 165.1/165.1 kB 8.8 MB/s eta 0:00:00
Collecting kiwisolver>=1.3.1 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading kiwisolver-1.4.7-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl.metadata (6.3 kB)
Collecting pyparsing>=2.3.1 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading pyparsing-3.2.0-py3-none-any.whl.metadata (5.0 kB)
Collecting python-dateutil>=2.7 (from matplotlib>=3.5.1->-r requirements.txt (line 3))
  Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Requirement already satisfied: networkx>=2.8 in /opt/conda/lib/python3.10/site-packages (from scikit-image>=0.20->scikit-image[data]>=0.20->-r requirements.txt (line 5)) (3.1)
Collecting imageio>=2.33 (from scikit-image>=0.20->scikit-image[data]>=0.20->-r requirements.txt (line 5))
  Downloading imageio-2.36.1-py3-none-any.whl.metadata (5.2 kB)
Collecting tifffile>=2022.8.12 (from scikit-image>=0.20->scikit-image[data]>=0.20->-r requirements.txt (line 5))
  Downloading tifffile-2024.9.20-py3-none-any.whl.metadata (32 kB)
Collecting lazy-loader>=0.4 (from scikit-image>=0.20->scikit-image[data]>=0.20->-r requirements.txt (line 5))
  Downloading lazy_loader-0.4-py3-none-any.whl.metadata (7.6 kB)
Requirement already satisfied: filelock in /opt/conda/lib/python3.10/site-packages (from torch>=2.0.1->-r requirements.txt (line 7)) (3.13.1)
Requirement already satisfied: typing-extensions>=4.8.0 in /opt/conda/lib/python3.10/site-packages (from torch>=2.0.1->-r requirements.txt (line 7)) (4.9.0)
Requirement already satisfied: sympy in /opt/conda/lib/python3.10/site-packages (from torch>=2.0.1->-r requirements.txt (line 7)) (1.12)
Requirement already satisfied: fsspec in /opt/conda/lib/python3.10/site-packages (from torch>=2.0.1->-r requirements.txt (line 7)) (2024.2.0)
Collecting lightning-utilities>=0.8.0 (from torchmetrics->-r requirements.txt (line 9))
  Downloading lightning_utilities-0.11.9-py3-none-any.whl.metadata (5.2 kB)
Collecting absl-py>=0.4 (from tensorboard->-r requirements.txt (line 10))
  Downloading absl_py-2.1.0-py3-none-any.whl.metadata (2.3 kB)
Collecting grpcio>=1.48.2 (from tensorboard->-r requirements.txt (line 10))
  Downloading grpcio-1.68.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.9 kB)
Collecting markdown>=2.6.8 (from tensorboard->-r requirements.txt (line 10))
  Downloading Markdown-3.7-py3-none-any.whl.metadata (7.0 kB)
Collecting protobuf!=4.24.0,>=3.19.6 (from tensorboard->-r requirements.txt (line 10))
  Downloading protobuf-5.29.1-cp38-abi3-manylinux2014_x86_64.whl.metadata (592 bytes)
Requirement already satisfied: six>1.9 in /opt/conda/lib/python3.10/site-packages (from tensorboard->-r requirements.txt (line 10)) (1.16.0)
Collecting tensorboard-data-server<0.8.0,>=0.7.0 (from tensorboard->-r requirements.txt (line 10))
  Downloading tensorboard_data_server-0.7.2-py3-none-manylinux_2_31_x86_64.whl.metadata (1.1 kB)
Collecting werkzeug>=1.0.1 (from tensorboard->-r requirements.txt (line 10))
  Downloading werkzeug-3.1.3-py3-none-any.whl.metadata (3.7 kB)
Requirement already satisfied: pyyaml in /opt/conda/lib/python3.10/site-packages (from timm->-r requirements.txt (line 13)) (6.0.1)
Collecting huggingface_hub (from timm->-r requirements.txt (line 13))
  Downloading huggingface_hub-0.26.5-py3-none-any.whl.metadata (13 kB)
Collecting safetensors (from timm->-r requirements.txt (line 13))
  Downloading safetensors-0.4.5-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.8 kB)
Collecting joblib>=1.2.0 (from scikit-learn>=0.18->-r requirements.txt (line 14))
  Downloading joblib-1.4.2-py3-none-any.whl.metadata (5.4 kB)
Collecting threadpoolctl>=3.1.0 (from scikit-learn>=0.18->-r requirements.txt (line 14))
  Downloading threadpoolctl-3.5.0-py3-none-any.whl.metadata (13 kB)
Requirement already satisfied: pytz>=2020.1 in /opt/conda/lib/python3.10/site-packages (from pandas>=1.3.5->-r requirements.txt (line 15)) (2023.3.post1)
Collecting tzdata>=2022.7 (from pandas>=1.3.5->-r requirements.txt (line 15))
  Downloading tzdata-2024.2-py2.py3-none-any.whl.metadata (1.4 kB)
Collecting regex!=2019.12.17 (from transformers>=4.36.2->-r requirements.txt (line 17))
  Downloading regex-2024.11.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (40 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.5/40.5 kB 1.3 MB/s eta 0:00:00
Requirement already satisfied: requests in /opt/conda/lib/python3.10/site-packages (from transformers>=4.36.2->-r requirements.txt (line 17)) (2.31.0)
Collecting tokenizers<0.22,>=0.21 (from transformers>=4.36.2->-r requirements.txt (line 17))
  Downloading tokenizers-0.21.0-cp39-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.7 kB)
Requirement already satisfied: psutil in /opt/conda/lib/python3.10/site-packages (from accelerate>=0.26.0->-r requirements.txt (line 18)) (5.9.0)
Collecting termcolor (from fire->-r requirements.txt (line 21))
  Downloading termcolor-2.5.0-py3-none-any.whl.metadata (6.1 kB)
Collecting tensorboardx (from acsconv->-r requirements.txt (line 24))
  Downloading tensorboardX-2.6.2.2-py2.py3-none-any.whl.metadata (5.8 kB)
Requirement already satisfied: certifi>=2023.7.22 in /opt/conda/lib/python3.10/site-packages (from kaggle->-r requirements.txt (line 25)) (2024.2.2)
Collecting python-slugify (from kaggle->-r requirements.txt (line 25))
  Downloading python_slugify-8.0.4-py2.py3-none-any.whl.metadata (8.5 kB)
Requirement already satisfied: urllib3 in /opt/conda/lib/python3.10/site-packages (from kaggle->-r requirements.txt (line 25)) (2.1.0)
Collecting bleach (from kaggle->-r requirements.txt (line 25))
  Downloading bleach-6.2.0-py3-none-any.whl.metadata (30 kB)
Collecting datasets>=2.0.0 (from evaluate->-r requirements.txt (line 26))
  Downloading datasets-3.2.0-py3-none-any.whl.metadata (20 kB)
Collecting dill (from evaluate->-r requirements.txt (line 26))
  Downloading dill-0.3.9-py3-none-any.whl.metadata (10 kB)
Collecting xxhash (from evaluate->-r requirements.txt (line 26))
  Downloading xxhash-3.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (12 kB)
Collecting multiprocess (from evaluate->-r requirements.txt (line 26))
  Downloading multiprocess-0.70.17-py310-none-any.whl.metadata (7.2 kB)
Collecting pooch>=1.6.0 (from scikit-image[data]>=0.20->-r requirements.txt (line 5))
  Downloading pooch-1.8.2-py3-none-any.whl.metadata (10 kB)
Collecting pyarrow>=15.0.0 (from datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading pyarrow-18.1.0-cp310-cp310-manylinux_2_28_x86_64.whl.metadata (3.3 kB)
Collecting dill (from evaluate->-r requirements.txt (line 26))
  Downloading dill-0.3.8-py3-none-any.whl.metadata (10 kB)
Collecting requests (from transformers>=4.36.2->-r requirements.txt (line 17))
  Downloading requests-2.32.3-py3-none-any.whl.metadata (4.6 kB)
Collecting tqdm>=4.62.3 (from -r requirements.txt (line 16))
  Downloading tqdm-4.67.1-py3-none-any.whl.metadata (57 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.7/57.7 kB 4.7 MB/s eta 0:00:00
Collecting multiprocess (from evaluate->-r requirements.txt (line 26))
  Downloading multiprocess-0.70.16-py310-none-any.whl.metadata (7.2 kB)
Collecting aiohttp (from datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading aiohttp-3.11.10-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (7.7 kB)
Collecting anyio (from httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1))
  Downloading anyio-4.7.0-py3-none-any.whl.metadata (4.7 kB)
Collecting httpcore==1.* (from httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1))
  Downloading httpcore-1.0.7-py3-none-any.whl.metadata (21 kB)
Requirement already satisfied: idna in /opt/conda/lib/python3.10/site-packages (from httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1)) (3.4)
Collecting h11<0.15,>=0.13 (from httpcore==1.*->httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1))
  Downloading h11-0.14.0-py3-none-any.whl.metadata (8.2 kB)
Collecting comm>=0.1.1 (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1))
  Downloading comm-0.2.2-py3-none-any.whl.metadata (3.7 kB)
Collecting debugpy>=1.6.5 (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1))
  Downloading debugpy-1.8.9-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.1 kB)
Requirement already satisfied: ipython>=7.23.1 in /opt/conda/lib/python3.10/site-packages (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (8.20.0)
Collecting jupyter-client>=6.1.12 (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_client-8.6.3-py3-none-any.whl.metadata (8.3 kB)
Requirement already satisfied: matplotlib-inline>=0.1 in /opt/conda/lib/python3.10/site-packages (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.1.6)
Collecting nest-asyncio (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1))
  Downloading nest_asyncio-1.6.0-py3-none-any.whl.metadata (2.8 kB)
Collecting pyzmq>=24 (from ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1))
  Downloading pyzmq-26.2.0-cp310-cp310-manylinux_2_28_x86_64.whl.metadata (6.2 kB)
Requirement already satisfied: MarkupSafe>=2.0 in /opt/conda/lib/python3.10/site-packages (from jinja2>=3.0.3->jupyterlab->-r requirements.txt (line 1)) (2.1.3)
Requirement already satisfied: platformdirs>=2.5 in /opt/conda/lib/python3.10/site-packages (from jupyter-core->jupyterlab->-r requirements.txt (line 1)) (3.10.0)
Collecting argon2-cffi>=21.1 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading argon2_cffi-23.1.0-py3-none-any.whl.metadata (5.2 kB)
Collecting jupyter-events>=0.9.0 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_events-0.10.0-py3-none-any.whl.metadata (5.9 kB)
Collecting jupyter-server-terminals>=0.4.4 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading jupyter_server_terminals-0.5.3-py3-none-any.whl.metadata (5.6 kB)
Collecting nbconvert>=6.4.4 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading nbconvert-7.16.4-py3-none-any.whl.metadata (8.5 kB)
Collecting nbformat>=5.3.0 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading nbformat-5.10.4-py3-none-any.whl.metadata (3.6 kB)
Collecting overrides>=5.0 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading overrides-7.7.0-py3-none-any.whl.metadata (5.8 kB)
Collecting prometheus-client>=0.9 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading prometheus_client-0.21.1-py3-none-any.whl.metadata (1.8 kB)
Collecting send2trash>=1.8.2 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading Send2Trash-1.8.3-py3-none-any.whl.metadata (4.0 kB)
Collecting terminado>=0.8.3 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading terminado-0.18.1-py3-none-any.whl.metadata (5.8 kB)
Collecting websocket-client>=1.7 (from jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading websocket_client-1.8.0-py3-none-any.whl.metadata (8.0 kB)
Collecting babel>=2.10 (from jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1))
  Downloading babel-2.16.0-py3-none-any.whl.metadata (1.5 kB)
Collecting json5>=0.9.0 (from jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1))
  Downloading json5-0.10.0-py3-none-any.whl.metadata (34 kB)
Requirement already satisfied: jsonschema>=4.18.0 in /opt/conda/lib/python3.10/site-packages (from jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1)) (4.19.2)
Requirement already satisfied: charset-normalizer<4,>=2 in /opt/conda/lib/python3.10/site-packages (from requests->transformers>=4.36.2->-r requirements.txt (line 17)) (2.0.4)
Collecting webencodings (from bleach->kaggle->-r requirements.txt (line 25))
  Downloading webencodings-0.5.1-py2.py3-none-any.whl.metadata (2.1 kB)
Collecting text-unidecode>=1.3 (from python-slugify->kaggle->-r requirements.txt (line 25))
  Downloading text_unidecode-1.3-py2.py3-none-any.whl.metadata (2.4 kB)
Requirement already satisfied: mpmath>=0.19 in /opt/conda/lib/python3.10/site-packages (from sympy->torch>=2.0.1->-r requirements.txt (line 7)) (1.3.0)
Collecting aiohappyeyeballs>=2.3.0 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading aiohappyeyeballs-2.4.4-py3-none-any.whl.metadata (6.1 kB)
Collecting aiosignal>=1.1.2 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading aiosignal-1.3.1-py3-none-any.whl.metadata (4.0 kB)
Collecting async-timeout<6.0,>=4.0 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading async_timeout-5.0.1-py3-none-any.whl.metadata (5.1 kB)
Requirement already satisfied: attrs>=17.3.0 in /opt/conda/lib/python3.10/site-packages (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26)) (23.1.0)
Collecting frozenlist>=1.1.1 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading frozenlist-1.5.0-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)
Collecting multidict<7.0,>=4.5 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading multidict-6.1.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.0 kB)
Collecting propcache>=0.2.0 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading propcache-0.2.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (9.2 kB)
Collecting yarl<2.0,>=1.17.0 (from aiohttp->datasets>=2.0.0->evaluate->-r requirements.txt (line 26))
  Downloading yarl-1.18.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (69 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 69.2/69.2 kB 4.8 MB/s eta 0:00:00
Requirement already satisfied: exceptiongroup>=1.0.2 in /opt/conda/lib/python3.10/site-packages (from anyio->httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1)) (1.2.0)
Collecting sniffio>=1.1 (from anyio->httpx>=0.25.0->jupyterlab->-r requirements.txt (line 1))
  Downloading sniffio-1.3.1-py3-none-any.whl.metadata (3.9 kB)
Collecting argon2-cffi-bindings (from argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading argon2_cffi_bindings-21.2.0-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.7 kB)
Requirement already satisfied: decorator in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (5.1.1)
Requirement already satisfied: jedi>=0.16 in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.18.1)
Requirement already satisfied: prompt-toolkit<3.1.0,>=3.0.41 in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (3.0.43)
Requirement already satisfied: pygments>=2.4.0 in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (2.15.1)
Requirement already satisfied: stack-data in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.2.0)
Requirement already satisfied: pexpect>4.3 in /opt/conda/lib/python3.10/site-packages (from ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (4.8.0)
Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /opt/conda/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1)) (2023.7.1)
Requirement already satisfied: referencing>=0.28.4 in /opt/conda/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1)) (0.30.2)
Requirement already satisfied: rpds-py>=0.7.1 in /opt/conda/lib/python3.10/site-packages (from jsonschema>=4.18.0->jupyterlab-server<3,>=2.27.1->jupyterlab->-r requirements.txt (line 1)) (0.10.6)
Collecting python-json-logger>=2.0.4 (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading python_json_logger-2.0.7-py3-none-any.whl.metadata (6.5 kB)
Collecting rfc3339-validator (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading rfc3339_validator-0.1.4-py2.py3-none-any.whl.metadata (1.5 kB)
Collecting rfc3986-validator>=0.1.1 (from jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading rfc3986_validator-0.1.1-py2.py3-none-any.whl.metadata (1.7 kB)
Requirement already satisfied: beautifulsoup4 in /opt/conda/lib/python3.10/site-packages (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (4.12.2)
Collecting defusedxml (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading defusedxml-0.7.1-py2.py3-none-any.whl.metadata (32 kB)
Collecting jupyterlab-pygments (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading jupyterlab_pygments-0.3.0-py3-none-any.whl.metadata (4.4 kB)
Collecting mistune<4,>=2.0.3 (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading mistune-3.0.2-py3-none-any.whl.metadata (1.7 kB)
Collecting nbclient>=0.5.0 (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading nbclient-0.10.1-py3-none-any.whl.metadata (8.2 kB)
Collecting pandocfilters>=1.4.1 (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading pandocfilters-1.5.1-py2.py3-none-any.whl.metadata (9.0 kB)
Collecting tinycss2 (from nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading tinycss2-1.4.0-py3-none-any.whl.metadata (3.0 kB)
Collecting fastjsonschema>=2.15 (from nbformat>=5.3.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading fastjsonschema-2.21.1-py3-none-any.whl.metadata (2.2 kB)
Requirement already satisfied: ptyprocess in /opt/conda/lib/python3.10/site-packages (from terminado>=0.8.3->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (0.7.0)
Requirement already satisfied: parso<0.9.0,>=0.8.0 in /opt/conda/lib/python3.10/site-packages (from jedi>=0.16->ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.8.3)
Collecting fqdn (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading fqdn-1.5.1-py3-none-any.whl.metadata (1.4 kB)
Collecting isoduration (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading isoduration-20.11.0-py3-none-any.whl.metadata (5.7 kB)
Requirement already satisfied: jsonpointer>1.13 in /opt/conda/lib/python3.10/site-packages (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (2.1)
Collecting uri-template (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading uri_template-1.3.0-py3-none-any.whl.metadata (8.8 kB)
Collecting webcolors>=1.11 (from jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading webcolors-24.11.1-py3-none-any.whl.metadata (2.2 kB)
Requirement already satisfied: wcwidth in /opt/conda/lib/python3.10/site-packages (from prompt-toolkit<3.1.0,>=3.0.41->ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.2.5)
Requirement already satisfied: cffi>=1.0.1 in /opt/conda/lib/python3.10/site-packages (from argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (1.16.0)
Requirement already satisfied: soupsieve>1.2 in /opt/conda/lib/python3.10/site-packages (from beautifulsoup4->nbconvert>=6.4.4->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (2.5)
Requirement already satisfied: executing in /opt/conda/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.8.3)
Requirement already satisfied: asttokens in /opt/conda/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (2.0.5)
Requirement already satisfied: pure-eval in /opt/conda/lib/python3.10/site-packages (from stack-data->ipython>=7.23.1->ipykernel>=6.5.0->jupyterlab->-r requirements.txt (line 1)) (0.2.2)
Requirement already satisfied: pycparser in /opt/conda/lib/python3.10/site-packages (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi>=21.1->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1)) (2.21)
Collecting arrow>=0.15.0 (from isoduration->jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading arrow-1.3.0-py3-none-any.whl.metadata (7.5 kB)
Collecting types-python-dateutil>=2.8.10 (from arrow>=0.15.0->isoduration->jsonschema[format-nongpl]>=4.18.0->jupyter-events>=0.9.0->jupyter-server<3,>=2.4.0->jupyterlab->-r requirements.txt (line 1))
  Downloading types_python_dateutil-2.9.0.20241206-py3-none-any.whl.metadata (2.1 kB)
Downloading jupyterlab-4.3.3-py3-none-any.whl (11.7 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.7/11.7 MB 55.1 MB/s eta 0:00:00
Downloading matplotlib-3.9.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (8.3 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.3/8.3 MB 60.3 MB/s eta 0:00:00
Downloading scipy-1.14.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (41.2 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 41.2/41.2 MB 26.8 MB/s eta 0:00:00
Downloading scikit_image-0.24.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (14.9 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 14.9/14.9 MB 40.1 MB/s eta 0:00:00
Downloading torchmetrics-1.6.0-py3-none-any.whl (926 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 926.4/926.4 kB 17.8 MB/s eta 0:00:00
Downloading tensorboard-2.18.0-py3-none-any.whl (5.5 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.5/5.5 MB 47.9 MB/s eta 0:00:00
Downloading torchsummary-1.5.1-py3-none-any.whl (2.8 kB)
Downloading einops-0.8.0-py3-none-any.whl (43 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.2/43.2 kB 2.9 MB/s eta 0:00:00
Downloading timm-1.0.12-py3-none-any.whl (2.4 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.4/2.4 MB 36.6 MB/s eta 0:00:00
Downloading scikit_learn-1.6.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (13.5 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.5/13.5 MB 50.1 MB/s eta 0:00:00
Downloading pandas-2.2.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (13.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.1/13.1 MB 49.6 MB/s eta 0:00:00
Downloading transformers-4.47.0-py3-none-any.whl (10.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 58.1 MB/s eta 0:00:00
Downloading accelerate-1.2.0-py3-none-any.whl (336 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 336.3/336.3 kB 10.0 MB/s eta 0:00:00
Downloading medmnist-3.0.2-py3-none-any.whl (25 kB)
Downloading monai-1.4.0-py3-none-any.whl (1.5 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.5/1.5 MB 30.8 MB/s eta 0:00:00
Downloading evaluate-0.4.3-py3-none-any.whl (84 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 84.0/84.0 kB 6.0 MB/s eta 0:00:00
Downloading absl_py-2.1.0-py3-none-any.whl (133 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 133.7/133.7 kB 8.0 MB/s eta 0:00:00
Downloading async_lru-2.0.4-py3-none-any.whl (6.1 kB)
Downloading contourpy-1.3.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (324 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 325.0/325.0 kB 13.4 MB/s eta 0:00:00
Downloading cycler-0.12.1-py3-none-any.whl (8.3 kB)
Downloading datasets-3.2.0-py3-none-any.whl (480 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 480.6/480.6 kB 14.2 MB/s eta 0:00:00
Downloading tqdm-4.67.1-py3-none-any.whl (78 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.5/78.5 kB 4.4 MB/s eta 0:00:00
Downloading dill-0.3.8-py3-none-any.whl (116 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 116.3/116.3 kB 6.6 MB/s eta 0:00:00
Downloading fonttools-4.55.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.6/4.6 MB 44.7 MB/s eta 0:00:00
Downloading grpcio-1.68.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.9 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.9/5.9 MB 51.5 MB/s eta 0:00:00
Downloading httpx-0.28.1-py3-none-any.whl (73 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 73.5/73.5 kB 5.8 MB/s eta 0:00:00
Downloading httpcore-1.0.7-py3-none-any.whl (78 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.6/78.6 kB 6.1 MB/s eta 0:00:00
Downloading huggingface_hub-0.26.5-py3-none-any.whl (447 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 447.8/447.8 kB 12.8 MB/s eta 0:00:00
Downloading imageio-2.36.1-py3-none-any.whl (315 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 315.4/315.4 kB 10.4 MB/s eta 0:00:00
Downloading ipykernel-6.29.5-py3-none-any.whl (117 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 117.2/117.2 kB 6.1 MB/s eta 0:00:00
Downloading joblib-1.4.2-py3-none-any.whl (301 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 301.8/301.8 kB 10.7 MB/s eta 0:00:00
Downloading jupyter_core-5.7.2-py3-none-any.whl (28 kB)
Downloading jupyter_lsp-2.2.5-py3-none-any.whl (69 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 69.1/69.1 kB 5.0 MB/s eta 0:00:00
Downloading jupyter_server-2.14.2-py3-none-any.whl (383 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 383.6/383.6 kB 13.1 MB/s eta 0:00:00
Downloading jupyterlab_server-2.27.3-py3-none-any.whl (59 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 59.7/59.7 kB 4.4 MB/s eta 0:00:00
Downloading kiwisolver-1.4.7-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (1.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 38.2 MB/s eta 0:00:00
Downloading lazy_loader-0.4-py3-none-any.whl (12 kB)
Downloading lightning_utilities-0.11.9-py3-none-any.whl (28 kB)
Downloading Markdown-3.7-py3-none-any.whl (106 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.3/106.3 kB 8.4 MB/s eta 0:00:00
Downloading multiprocess-0.70.16-py310-none-any.whl (134 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 134.8/134.8 kB 6.3 MB/s eta 0:00:00
Downloading notebook_shim-0.2.4-py3-none-any.whl (13 kB)
Downloading pooch-1.8.2-py3-none-any.whl (64 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.6/64.6 kB 3.4 MB/s eta 0:00:00
Downloading protobuf-5.29.1-cp38-abi3-manylinux2014_x86_64.whl (319 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 319.7/319.7 kB 11.7 MB/s eta 0:00:00
Downloading pyparsing-3.2.0-py3-none-any.whl (106 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.9/106.9 kB 5.3 MB/s eta 0:00:00
Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 229.9/229.9 kB 12.3 MB/s eta 0:00:00
Downloading regex-2024.11.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (781 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 781.7/781.7 kB 19.8 MB/s eta 0:00:00
Downloading requests-2.32.3-py3-none-any.whl (64 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.9/64.9 kB 4.9 MB/s eta 0:00:00
Downloading safetensors-0.4.5-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (435 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 435.0/435.0 kB 14.7 MB/s eta 0:00:00
Downloading tensorboard_data_server-0.7.2-py3-none-manylinux_2_31_x86_64.whl (6.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.6/6.6 MB 49.3 MB/s eta 0:00:00
Downloading threadpoolctl-3.5.0-py3-none-any.whl (18 kB)
Downloading tifffile-2024.9.20-py3-none-any.whl (228 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 228.2/228.2 kB 12.4 MB/s eta 0:00:00
Downloading tokenizers-0.21.0-cp39-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.0 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.0/3.0 MB 41.3 MB/s eta 0:00:00
Downloading tornado-6.4.2-cp38-abi3-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (437 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 437.2/437.2 kB 14.7 MB/s eta 0:00:00
Downloading tzdata-2024.2-py2.py3-none-any.whl (346 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 346.6/346.6 kB 12.8 MB/s eta 0:00:00
Downloading werkzeug-3.1.3-py3-none-any.whl (224 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 224.5/224.5 kB 10.6 MB/s eta 0:00:00
Downloading bleach-6.2.0-py3-none-any.whl (163 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 163.4/163.4 kB 10.9 MB/s eta 0:00:00
Downloading python_slugify-8.0.4-py2.py3-none-any.whl (10 kB)
Downloading tensorboardX-2.6.2.2-py2.py3-none-any.whl (101 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 101.7/101.7 kB 7.6 MB/s eta 0:00:00
Downloading termcolor-2.5.0-py3-none-any.whl (7.8 kB)
Downloading xxhash-3.5.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (194 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 194.1/194.1 kB 9.8 MB/s eta 0:00:00
Downloading aiohttp-3.11.10-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 41.8 MB/s eta 0:00:00
Downloading anyio-4.7.0-py3-none-any.whl (93 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 93.1/93.1 kB 6.9 MB/s eta 0:00:00
Downloading argon2_cffi-23.1.0-py3-none-any.whl (15 kB)
Downloading babel-2.16.0-py3-none-any.whl (9.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 9.6/9.6 MB 58.0 MB/s eta 0:00:00
Downloading comm-0.2.2-py3-none-any.whl (7.2 kB)
Downloading debugpy-1.8.9-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.6 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.6/3.6 MB 47.2 MB/s eta 0:00:00
Downloading json5-0.10.0-py3-none-any.whl (34 kB)
Downloading jupyter_client-8.6.3-py3-none-any.whl (106 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 106.1/106.1 kB 7.9 MB/s eta 0:00:00
Downloading jupyter_events-0.10.0-py3-none-any.whl (18 kB)
Downloading jupyter_server_terminals-0.5.3-py3-none-any.whl (13 kB)
Downloading nbconvert-7.16.4-py3-none-any.whl (257 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 257.4/257.4 kB 17.8 MB/s eta 0:00:00
Downloading nbformat-5.10.4-py3-none-any.whl (78 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.5/78.5 kB 5.7 MB/s eta 0:00:00
Downloading overrides-7.7.0-py3-none-any.whl (17 kB)
Downloading prometheus_client-0.21.1-py3-none-any.whl (54 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 54.7/54.7 kB 4.1 MB/s eta 0:00:00
Downloading pyarrow-18.1.0-cp310-cp310-manylinux_2_28_x86_64.whl (40.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.1/40.1 MB 35.6 MB/s eta 0:00:00
Downloading pyzmq-26.2.0-cp310-cp310-manylinux_2_28_x86_64.whl (868 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 868.8/868.8 kB 38.0 MB/s eta 0:00:00
Downloading Send2Trash-1.8.3-py3-none-any.whl (18 kB)
Downloading terminado-0.18.1-py3-none-any.whl (14 kB)
Downloading text_unidecode-1.3-py2.py3-none-any.whl (78 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.2/78.2 kB 6.7 MB/s eta 0:00:00
Downloading websocket_client-1.8.0-py3-none-any.whl (58 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.8/58.8 kB 4.6 MB/s eta 0:00:00
Downloading nest_asyncio-1.6.0-py3-none-any.whl (5.2 kB)
Downloading webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
Downloading aiohappyeyeballs-2.4.4-py3-none-any.whl (14 kB)
Downloading aiosignal-1.3.1-py3-none-any.whl (7.6 kB)
Downloading async_timeout-5.0.1-py3-none-any.whl (6.2 kB)
Downloading fastjsonschema-2.21.1-py3-none-any.whl (23 kB)
Downloading frozenlist-1.5.0-cp310-cp310-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (241 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 241.9/241.9 kB 19.2 MB/s eta 0:00:00
Downloading h11-0.14.0-py3-none-any.whl (58 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 kB 5.1 MB/s eta 0:00:00
Downloading mistune-3.0.2-py3-none-any.whl (47 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 48.0/48.0 kB 3.8 MB/s eta 0:00:00
Downloading multidict-6.1.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (124 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 124.6/124.6 kB 11.2 MB/s eta 0:00:00
Downloading nbclient-0.10.1-py3-none-any.whl (25 kB)
Downloading pandocfilters-1.5.1-py2.py3-none-any.whl (8.7 kB)
Downloading propcache-0.2.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (205 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 205.1/205.1 kB 17.4 MB/s eta 0:00:00
Downloading python_json_logger-2.0.7-py3-none-any.whl (8.1 kB)
Downloading rfc3986_validator-0.1.1-py2.py3-none-any.whl (4.2 kB)
Downloading sniffio-1.3.1-py3-none-any.whl (10 kB)
Downloading yarl-1.18.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (319 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 319.7/319.7 kB 22.0 MB/s eta 0:00:00
Downloading argon2_cffi_bindings-21.2.0-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (86 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 86.2/86.2 kB 6.6 MB/s eta 0:00:00
Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Downloading jupyterlab_pygments-0.3.0-py3-none-any.whl (15 kB)
Downloading rfc3339_validator-0.1.4-py2.py3-none-any.whl (3.5 kB)
Downloading tinycss2-1.4.0-py3-none-any.whl (26 kB)
Downloading webcolors-24.11.1-py3-none-any.whl (14 kB)
Downloading fqdn-1.5.1-py3-none-any.whl (9.1 kB)
Downloading isoduration-20.11.0-py3-none-any.whl (11 kB)
Downloading uri_template-1.3.0-py3-none-any.whl (11 kB)
Downloading arrow-1.3.0-py3-none-any.whl (66 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.4/66.4 kB 5.1 MB/s eta 0:00:00
Downloading types_python_dateutil-2.9.0.20241206-py3-none-any.whl (14 kB)
Building wheels for collected packages: fire, acsconv, kaggle
  Building wheel for fire (setup.py): started
  Building wheel for fire (setup.py): finished with status 'done'
  Created wheel for fire: filename=fire-0.7.0-py3-none-any.whl size=114247 sha256=94635152dc3455ecac22e66e2ec4695c23b7e4169f4b2ff5caf165f49efb37a6
  Stored in directory: /tmp/cache/pip/wheels/19/39/2f/2d3cadc408a8804103f1c34ddd4b9f6a93497b11fa96fe738e
  Building wheel for acsconv (setup.py): started
  Building wheel for acsconv (setup.py): finished with status 'done'
  Created wheel for acsconv: filename=ACSConv-0.1.1-py3-none-any.whl size=24182 sha256=f78ee89798f4a1b3ade66f60733dd1a56b5d838356e84254f0466e85dfbf34ca
  Stored in directory: /tmp/cache/pip/wheels/0e/b1/ef/43fe6db442ef60fc6a8c7f98f330672eeca58463ebef542e62
  Building wheel for kaggle (setup.py): started
  Building wheel for kaggle (setup.py): finished with status 'done'
  Created wheel for kaggle: filename=kaggle-1.6.17-py3-none-any.whl size=105786 sha256=d0b94f085333ec8f74be90a56f21b82650a05ccdfc01933d31b693444b4aeb28
  Stored in directory: /tmp/cache/pip/wheels/9f/af/22/bf406f913dc7506a485e60dce8143741abd0a92a19337d83a3
Successfully built fire acsconv kaggle
Installing collected packages: webencodings, torchsummary, text-unidecode, fastjsonschema, xxhash, werkzeug, websocket-client, webcolors, uri-template, tzdata, types-python-dateutil, tqdm, tornado, tinycss2, tifffile, threadpoolctl, termcolor, tensorboard-data-server, sniffio, send2trash, scipy, safetensors, rfc3986-validator, rfc3339-validator, requests, regex, pyzmq, python-slugify, python-json-logger, python-dateutil, pyparsing, pyarrow, protobuf, propcache, prometheus-client, pandocfilters, overrides, nest-asyncio, multidict, mistune, markdown, lightning-utilities, lazy-loader, kiwisolver, jupyterlab-pygments, jupyter-core, json5, joblib, imageio, h11, grpcio, frozenlist, fqdn, fonttools, einops, dill, defusedxml, debugpy, cycler, contourpy, comm, bleach, babel, async-timeout, async-lru, aiohappyeyeballs, absl-py, yarl, terminado, tensorboardx, tensorboard, scikit-learn, scikit-image, pooch, pandas, multiprocess, matplotlib, kaggle, jupyter-client, huggingface_hub, httpcore, fire, arrow, argon2-cffi-bindings, anyio, aiosignal, torchmetrics, tokenizers, monai, jupyter-server-terminals, isoduration, httpx, argon2-cffi, aiohttp, accelerate, transformers, timm, nbformat, medmnist, ipykernel, acsconv, nbclient, jupyter-events, datasets, nbconvert, evaluate, jupyter-server, notebook-shim, jupyterlab-server, jupyter-lsp, jupyterlab
  Attempting uninstall: tqdm
    Found existing installation: tqdm 4.65.0
    Uninstalling tqdm-4.65.0:
      Successfully uninstalled tqdm-4.65.0
  Attempting uninstall: requests
    Found existing installation: requests 2.31.0
    Uninstalling requests-2.31.0:
      Successfully uninstalled requests-2.31.0
Successfully installed absl-py-2.1.0 accelerate-1.2.0 acsconv-0.1.1 aiohappyeyeballs-2.4.4 aiohttp-3.11.10 aiosignal-1.3.1 anyio-4.7.0 argon2-cffi-23.1.0 argon2-cffi-bindings-21.2.0 arrow-1.3.0 async-lru-2.0.4 async-timeout-5.0.1 babel-2.16.0 bleach-6.2.0 comm-0.2.2 contourpy-1.3.1 cycler-0.12.1 datasets-3.2.0 debugpy-1.8.9 defusedxml-0.7.1 dill-0.3.8 einops-0.8.0 evaluate-0.4.3 fastjsonschema-2.21.1 fire-0.7.0 fonttools-4.55.3 fqdn-1.5.1 frozenlist-1.5.0 grpcio-1.68.1 h11-0.14.0 httpcore-1.0.7 httpx-0.28.1 huggingface_hub-0.26.5 imageio-2.36.1 ipykernel-6.29.5 isoduration-20.11.0 joblib-1.4.2 json5-0.10.0 jupyter-client-8.6.3 jupyter-core-5.7.2 jupyter-events-0.10.0 jupyter-lsp-2.2.5 jupyter-server-2.14.2 jupyter-server-terminals-0.5.3 jupyterlab-4.3.3 jupyterlab-pygments-0.3.0 jupyterlab-server-2.27.3 kaggle-1.6.17 kiwisolver-1.4.7 lazy-loader-0.4 lightning-utilities-0.11.9 markdown-3.7 matplotlib-3.9.3 medmnist-3.0.2 mistune-3.0.2 monai-1.4.0 multidict-6.1.0 multiprocess-0.70.16 nbclient-0.10.1 nbconvert-7.16.4 nbformat-5.10.4 nest-asyncio-1.6.0 notebook-shim-0.2.4 overrides-7.7.0 pandas-2.2.3 pandocfilters-1.5.1 pooch-1.8.2 prometheus-client-0.21.1 propcache-0.2.1 protobuf-5.29.1 pyarrow-18.1.0 pyparsing-3.2.0 python-dateutil-2.9.0.post0 python-json-logger-2.0.7 python-slugify-8.0.4 pyzmq-26.2.0 regex-2024.11.6 requests-2.32.3 rfc3339-validator-0.1.4 rfc3986-validator-0.1.1 safetensors-0.4.5 scikit-image-0.24.0 scikit-learn-1.6.0 scipy-1.14.1 send2trash-1.8.3 sniffio-1.3.1 tensorboard-2.18.0 tensorboard-data-server-0.7.2 tensorboardx-2.6.2.2 termcolor-2.5.0 terminado-0.18.1 text-unidecode-1.3 threadpoolctl-3.5.0 tifffile-2024.9.20 timm-1.0.12 tinycss2-1.4.0 tokenizers-0.21.0 torchmetrics-1.6.0 torchsummary-1.5.1 tornado-6.4.2 tqdm-4.67.1 transformers-4.47.0 types-python-dateutil-2.9.0.20241206 tzdata-2024.2 uri-template-1.3.0 webcolors-24.11.1 webencodings-0.5.1 websocket-client-1.8.0 werkzeug-3.1.3 xxhash-3.5.0 yarl-1.18.3
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
useradd: warning: the home directory /mnt/hdd/marina already exists.
useradd: Not copying any file from skel directory into it.
GPU 0: NVIDIA GeForce RTX 2080 Ti (UUID: GPU-c5f27bee-5c06-3751-008a-30344aee726c)
[I 2024-12-11 17:15:04.469 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2024-12-11 17:15:04.472 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2024-12-11 17:15:04.475 ServerApp] jupyterlab | extension was successfully linked.
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[I 2024-12-11 17:15:04.647 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-12-11 17:15:04.659 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-12-11 17:15:04.661 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2024-12-11 17:15:04.662 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2024-12-11 17:15:04.663 LabApp] JupyterLab extension loaded from /opt/conda/lib/python3.10/site-packages/jupyterlab
[I 2024-12-11 17:15:04.663 LabApp] JupyterLab application directory is /opt/conda/share/jupyter/lab
[I 2024-12-11 17:15:04.663 LabApp] Extension Manager is 'pypi'.
[I 2024-12-11 17:15:04.704 ServerApp] jupyterlab | extension was successfully loaded.
[I 2024-12-11 17:15:04.705 ServerApp] Serving notebooks from local directory: /mnt/hdd/marina/workspace/2D-to-3D-Pretraining
[I 2024-12-11 17:15:04.705 ServerApp] Jupyter Server 2.14.2 is running at:
[I 2024-12-11 17:15:04.705 ServerApp] http://85a6b920636a:9999/lab?token=4bc64cb8403a466a7ad156b85a06d856efd4b464243eac1d
[I 2024-12-11 17:15:04.705 ServerApp]     http://127.0.0.1:9999/lab?token=4bc64cb8403a466a7ad156b85a06d856efd4b464243eac1d
[I 2024-12-11 17:15:04.705 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 2024-12-11 17:15:04.710 ServerApp] No web browser found: Error('could not locate runnable browser').
[C 2024-12-11 17:15:04.710 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///mnt/hdd/marina/.local/share/jupyter/runtime/jpserver-345-open.html
    Or copy and paste one of these URLs:
        http://85a6b920636a:9999/lab?token=4bc64cb8403a466a7ad156b85a06d856efd4b464243eac1d
        http://127.0.0.1:9999/lab?token=4bc64cb8403a466a7ad156b85a06d856efd4b464243eac1d
[I 2024-12-11 17:15:04.731 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[W 2024-12-11 17:15:12.658 LabApp] Could not determine jupyterlab build status without nodejs
[I 2024-12-11 17:15:15.362 ServerApp] Kernel started: ef334d12-069f-4d80-90ec-746082a5dc5a
[I 2024-12-11 17:15:15.370 ServerApp] Kernel started: 0fee1cc3-6834-468c-963a-b25b9d00c577
[I 2024-12-11 17:15:15.589 ServerApp] Kernel started: 51324cbb-70b7-49f3-adb3-2ea7f5802806
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[I 2024-12-11 17:15:15.780 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[I 2024-12-11 17:15:15.813 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:1fb1b39b-054e-4f17-ae52-1f2b5cd669ef
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[I 2024-12-11 17:15:15.902 ServerApp] Connecting to kernel ef334d12-069f-4d80-90ec-746082a5dc5a.
[I 2024-12-11 17:15:15.924 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
[I 2024-12-11 17:15:16.328 ServerApp] Kernel started: 42c9f3ec-01d7-419c-a976-11780b1cc9fd
/opt/conda/lib/python3.10/site-packages/IPython/paths.py:69: UserWarning: IPython parent '/mnt/hdd/marina' is not a writable location, using a temp directory.
  warn("IPython parent '{0}' is not a writable location,"
[I 2024-12-11 17:15:16.904 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-11 17:15:16.932 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-11 17:15:16.963 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-11 17:15:16.999 ServerApp] Connecting to kernel 42c9f3ec-01d7-419c-a976-11780b1cc9fd.
[I 2024-12-11 17:17:15.762 ServerApp] Saving file at /MedMNIST_MONAI-3channels.ipynb
[I 2024-12-11 17:19:16.945 ServerApp] Saving file at /MedMNIST_MONAI-3channels.ipynb
[I 2024-12-11 17:21:19.006 ServerApp] Saving file at /MedMNIST_MONAI-3channels.ipynb
[E 2024-12-11 17:28:32.344 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-11 17:28:32.347 ServerApp] 405 CONNECT 193.149.189.126:7227 (@154.216.17.84) 3.74ms referer=None
[I 2024-12-11 17:59:59.992 ServerApp] Starting buffering for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:f673c3d6-a26c-4fd6-bab2-22b036d944cb
[I 2024-12-11 17:59:59.993 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:9b4ec9b8-60eb-4780-bb26-dcb61f937f95
[I 2024-12-11 17:59:59.994 ServerApp] Starting buffering for ef334d12-069f-4d80-90ec-746082a5dc5a:5d18ef66-e7c7-4b6a-9eb7-deddab7231e1
[I 2024-12-11 17:59:59.996 ServerApp] Starting buffering for 51324cbb-70b7-49f3-adb3-2ea7f5802806:b60a1003-906e-450d-a2e2-d5f4800321ba
[I 2024-12-11 18:38:52.290 ServerApp] 302 GET / (@167.94.146.50) 1.27ms
[I 2024-12-11 18:39:02.207 ServerApp] 302 GET / (@167.94.146.50) 0.84ms
[I 2024-12-11 18:39:02.592 ServerApp] Malformed HTTP message from 167.94.146.50: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-11 18:39:07.127 LabApp] 302 GET /lab (@167.94.146.50) 1.39ms
[I 2024-12-11 18:39:07.188 ServerApp] Malformed HTTP message from 167.94.146.50: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-11 18:39:07.225 ServerApp] 302 GET / (@167.94.146.50) 0.72ms
[I 2024-12-11 18:39:12.207 ServerApp] Malformed HTTP message from 167.94.146.50: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-11 18:39:12.269 ServerApp] 302 GET / (@167.94.146.50) 0.68ms
[E 2024-12-11 19:21:18.908 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-11 19:21:18.910 ServerApp] 405 CONNECT api.ip.pn:443 (@80.75.212.9) 1.99ms referer=None
[I 2024-12-11 20:27:05.144 ServerApp] Connecting to kernel ef334d12-069f-4d80-90ec-746082a5dc5a.
[I 2024-12-11 20:27:05.145 ServerApp] Restoring connection for ef334d12-069f-4d80-90ec-746082a5dc5a:5d18ef66-e7c7-4b6a-9eb7-deddab7231e1
[I 2024-12-11 20:27:05.173 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
[I 2024-12-11 20:27:05.173 ServerApp] Restoring connection for 0fee1cc3-6834-468c-963a-b25b9d00c577:9b4ec9b8-60eb-4780-bb26-dcb61f937f95
[I 2024-12-11 20:27:05.203 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-11 20:27:05.203 ServerApp] Restoring connection for 51324cbb-70b7-49f3-adb3-2ea7f5802806:b60a1003-906e-450d-a2e2-d5f4800321ba
[I 2024-12-11 20:27:05.227 ServerApp] Connecting to kernel 42c9f3ec-01d7-419c-a976-11780b1cc9fd.
[I 2024-12-11 20:27:05.227 ServerApp] Restoring connection for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:f673c3d6-a26c-4fd6-bab2-22b036d944cb
[I 2024-12-11 20:29:06.817 ServerApp] Saving file at /MedMNIST_MONAI-3channels.ipynb
[E 2024-12-11 22:25:43.117 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-11 22:25:43.118 ServerApp] 405 CONNECT 45.61.136.175:7227 (@178.215.236.52) 2.10ms referer=None
[I 2024-12-11 23:00:19.781 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:9b4ec9b8-60eb-4780-bb26-dcb61f937f95
[I 2024-12-11 23:00:19.782 ServerApp] Starting buffering for 51324cbb-70b7-49f3-adb3-2ea7f5802806:b60a1003-906e-450d-a2e2-d5f4800321ba
[I 2024-12-11 23:00:19.783 ServerApp] Starting buffering for ef334d12-069f-4d80-90ec-746082a5dc5a:5d18ef66-e7c7-4b6a-9eb7-deddab7231e1
[I 2024-12-11 23:00:19.784 ServerApp] Starting buffering for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:f673c3d6-a26c-4fd6-bab2-22b036d944cb
[E 2024-12-11 23:48:17.530 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-11 23:48:17.532 ServerApp] 405 CONNECT api.ip.pn:443 (@80.75.212.9) 1.94ms referer=None
[I 2024-12-12 00:01:01.633 ServerApp] Connecting to kernel ef334d12-069f-4d80-90ec-746082a5dc5a.
[I 2024-12-12 00:01:01.633 ServerApp] Restoring connection for ef334d12-069f-4d80-90ec-746082a5dc5a:5d18ef66-e7c7-4b6a-9eb7-deddab7231e1
[I 2024-12-12 00:01:01.661 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
[I 2024-12-12 00:01:01.662 ServerApp] Restoring connection for 0fee1cc3-6834-468c-963a-b25b9d00c577:9b4ec9b8-60eb-4780-bb26-dcb61f937f95
[I 2024-12-12 00:01:01.687 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-12 00:01:01.687 ServerApp] Restoring connection for 51324cbb-70b7-49f3-adb3-2ea7f5802806:b60a1003-906e-450d-a2e2-d5f4800321ba
[I 2024-12-12 00:01:01.711 ServerApp] Connecting to kernel 42c9f3ec-01d7-419c-a976-11780b1cc9fd.
[I 2024-12-12 00:01:01.712 ServerApp] Restoring connection for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:f673c3d6-a26c-4fd6-bab2-22b036d944cb
[I 2024-12-12 01:02:11.882 ServerApp] Starting buffering for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:f673c3d6-a26c-4fd6-bab2-22b036d944cb
[I 2024-12-12 01:02:11.883 ServerApp] Starting buffering for 51324cbb-70b7-49f3-adb3-2ea7f5802806:b60a1003-906e-450d-a2e2-d5f4800321ba
[I 2024-12-12 01:02:11.884 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:9b4ec9b8-60eb-4780-bb26-dcb61f937f95
[I 2024-12-12 01:02:11.885 ServerApp] Starting buffering for ef334d12-069f-4d80-90ec-746082a5dc5a:5d18ef66-e7c7-4b6a-9eb7-deddab7231e1
[E 2024-12-12 03:34:48.680 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-12 03:34:48.681 ServerApp] 405 CONNECT 185.65.245.140:7227 (@154.216.17.93) 1.92ms referer=None
[E 2024-12-12 04:11:25.681 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-12 04:11:25.682 ServerApp] 405 CONNECT api.ip.pn:443 (@80.75.212.9) 2.40ms referer=None
[I 2024-12-12 05:30:35.417 ServerApp] 302 GET / (@104.40.74.240) 1.80ms
[E 2024-12-12 06:08:13.319 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-12 06:08:13.321 ServerApp] 405 CONNECT 193.149.189.126:7227 (@94.156.167.76) 2.53ms referer=None
[I 2024-12-12 06:19:47.107 ServerApp] 302 GET / (@64.227.98.215) 0.89ms
[I 2024-12-12 06:19:47.258 LabApp] 302 GET /lab? (@64.227.98.215) 1.57ms
[E 2024-12-12 08:26:53.110 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-12 08:26:53.112 ServerApp] 405 CONNECT api.ip.pn:443 (@80.75.212.58) 2.21ms referer=None
[W 2024-12-12 08:33:02.526 ServerApp] 404 GET /aaa9 (@64.226.98.14) 14.73ms referer=None
[W 2024-12-12 08:33:04.288 ServerApp] 404 GET /aab8 (@64.226.98.14) 1.99ms referer=None
[I 2024-12-12 08:33:05.587 ServerApp] 302 GET / (@64.226.98.14) 0.72ms
[I 2024-12-12 08:46:10.228 ServerApp] 302 GET / (@195.3.223.52) 1.02ms
[W 2024-12-12 10:34:34.139 LabApp] Could not determine jupyterlab build status without nodejs
[I 2024-12-12 10:34:34.173 ServerApp] Connecting to kernel ef334d12-069f-4d80-90ec-746082a5dc5a.
[I 2024-12-12 10:34:34.205 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
[I 2024-12-12 10:34:34.235 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-12 10:34:34.291 ServerApp] Connecting to kernel 42c9f3ec-01d7-419c-a976-11780b1cc9fd.
[I 2024-12-12 10:34:34.315 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:1fd0c3e7-00e2-4b4e-9a35-f601df8ff727
[I 2024-12-12 10:34:34.326 ServerApp] Starting buffering for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:882b0154-3f1f-47c4-9a77-e47f72201b27
[I 2024-12-12 10:34:34.361 ServerApp] Starting buffering for ef334d12-069f-4d80-90ec-746082a5dc5a:a50efd31-d870-46c2-888d-ab328d661072
[I 2024-12-12 10:34:34.361 ServerApp] Starting buffering for 51324cbb-70b7-49f3-adb3-2ea7f5802806:bd090270-2cc5-4093-bebf-12044a225577
[I 2024-12-12 10:34:38.422 ServerApp] Connecting to kernel 0fee1cc3-6834-468c-963a-b25b9d00c577.
[I 2024-12-12 10:34:38.456 ServerApp] Connecting to kernel 42c9f3ec-01d7-419c-a976-11780b1cc9fd.
[I 2024-12-12 10:34:38.486 ServerApp] Connecting to kernel 51324cbb-70b7-49f3-adb3-2ea7f5802806.
[I 2024-12-12 10:34:38.513 ServerApp] Connecting to kernel ef334d12-069f-4d80-90ec-746082a5dc5a.
[E 2024-12-12 10:53:34.763 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-12 10:53:34.765 ServerApp] 405 CONNECT api.ip.pn:443 (@80.75.212.18) 1.92ms referer=None
[I 2024-12-12 11:01:45.809 ServerApp] Starting buffering for 0fee1cc3-6834-468c-963a-b25b9d00c577:c48d60c9-f396-4b53-bb2b-9b6b9882b86f
[I 2024-12-12 11:01:45.810 ServerApp] Starting buffering for 42c9f3ec-01d7-419c-a976-11780b1cc9fd:4b68dad7-0ead-4420-916b-b10848c19ab2
[I 2024-12-12 11:01:45.811 ServerApp] Starting buffering for 51324cbb-70b7-49f3-adb3-2ea7f5802806:ae28a6e4-f507-4976-bb27-a83521e69c7a
[I 2024-12-12 11:01:45.812 ServerApp] Starting buffering for ef334d12-069f-4d80-90ec-746082a5dc5a:3bcaa5a3-895c-4034-9dea-5fcfa578dc67
[I 2024-12-12 14:08:27.079 ServerApp] 302 GET / (@64.62.156.97) 0.93ms
[I 2024-12-12 14:08:33.162 ServerApp] 302 GET / (@64.62.156.100) 0.75ms
[I 2024-12-12 14:08:33.309 LabApp] 302 GET /lab? (@64.62.156.100) 1.06ms
[I 2024-12-12 16:13:32.809 ServerApp] Malformed HTTP message from 34.95.250.146: Malformed HTTP version in HTTP Request-Line: "\x85XIúf\x9b\x96)'Å\x92\xadè\x10\x1aD6Û/`FÃ"
[W 2024-12-12 16:13:42.854 ServerApp] 405 OPTIONS / (@34.95.250.146) 0.99ms referer=None
[I 2024-12-12 18:27:57.657 ServerApp] 302 GET / (@104.209.33.53) 0.93ms
[I 2024-12-13 03:56:59.190 ServerApp] 302 GET / (@172.245.40.162) 1.85ms
[I 2024-12-13 03:56:59.301 LabApp] 302 GET /lab? (@172.245.40.162) 1.60ms
[I 2024-12-13 07:49:18.018 ServerApp] 302 GET / (@74.82.47.4) 0.89ms
[I 2024-12-13 07:49:24.630 ServerApp] 302 GET / (@74.82.47.4) 0.87ms
[I 2024-12-13 07:49:24.776 LabApp] 302 GET /lab? (@74.82.47.4) 0.99ms
[E 2024-12-13 09:05:18.197 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-13 09:05:18.199 ServerApp] 405 CONNECT 185.65.245.140:7227 (@94.103.125.176) 2.12ms referer=None
[I 2024-12-13 09:47:32.955 ServerApp] 302 GET / (@139.144.31.57) 0.96ms
[I 2024-12-13 10:25:25.603 ServerApp] 302 GET / (@199.45.154.156) 0.79ms
[I 2024-12-13 10:25:37.772 ServerApp] 302 GET / (@199.45.154.156) 0.81ms
[I 2024-12-13 10:25:38.807 ServerApp] Malformed HTTP message from 199.45.154.156: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-13 10:25:43.858 LabApp] 302 GET /lab (@199.45.154.156) 0.98ms
[I 2024-12-13 10:25:44.711 ServerApp] Malformed HTTP message from 199.45.154.156: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-13 10:25:45.340 ServerApp] 302 GET / (@199.45.154.156) 0.71ms
[I 2024-12-13 10:25:53.911 ServerApp] Malformed HTTP message from 199.45.154.156: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-13 10:25:54.694 ServerApp] 302 GET / (@199.45.154.156) 0.69ms
[E 2024-12-13 12:11:31.783 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-13 12:11:31.784 ServerApp] 405 CONNECT 45.61.136.175:7227 (@45.95.147.164) 1.84ms referer=None
[E 2024-12-13 22:47:39.743 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-13 22:47:39.745 ServerApp] 405 CONNECT api.ip.pn:443 (@185.91.127.81) 2.63ms referer=None
[E 2024-12-14 00:31:52.240 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-14 00:31:52.241 ServerApp] 405 CONNECT api.ip.pn:443 (@193.68.89.3) 1.89ms referer=None
[E 2024-12-14 03:58:52.983 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-14 03:58:52.984 ServerApp] 405 CONNECT api.ip.pn:443 (@185.91.127.81) 2.18ms referer=None
[I 2024-12-14 06:54:58.765 ServerApp] 302 GET / (@64.62.156.22) 0.91ms
[I 2024-12-14 06:55:05.295 ServerApp] 302 GET / (@64.62.156.10) 0.75ms
[I 2024-12-14 06:55:05.442 LabApp] 302 GET /lab? (@64.62.156.10) 0.99ms
[I 2024-12-14 07:31:57.448 ServerApp] 302 GET / (@170.187.142.75) 0.83ms
[I 2024-12-14 07:31:57.556 LabApp] 302 GET /lab (@170.187.142.75) 0.98ms
[I 2024-12-14 07:33:20.139 ServerApp] 302 GET / (@206.81.13.187) 0.76ms
[I 2024-12-14 07:33:20.235 LabApp] 302 GET /lab (@206.81.13.187) 0.93ms
[I 2024-12-14 07:51:36.625 ServerApp] 302 GET / (@162.142.125.196) 0.82ms
[I 2024-12-14 07:51:49.681 ServerApp] 302 GET / (@162.142.125.196) 0.71ms
[I 2024-12-14 07:51:50.839 ServerApp] Malformed HTTP message from 162.142.125.196: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 07:51:59.295 LabApp] 302 GET /lab (@162.142.125.196) 1.05ms
[I 2024-12-14 07:52:01.349 ServerApp] Malformed HTTP message from 162.142.125.196: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 07:52:02.832 ServerApp] 302 GET / (@162.142.125.196) 0.70ms
[I 2024-12-14 07:52:14.442 ServerApp] Malformed HTTP message from 162.142.125.196: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 07:52:15.056 ServerApp] 302 GET / (@162.142.125.196) 0.69ms
[E 2024-12-14 08:56:30.421 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-14 08:56:30.422 ServerApp] 405 CONNECT api.ip.pn:443 (@193.68.89.3) 1.93ms referer=None
[E 2024-12-14 09:22:35.149 ServerApp] Uncaught exception in write_error
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1753, in _execute
        raise HTTPError(405)
    tornado.web.HTTPError: HTTP 405: Method Not Allowed
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1298, in send_error
        self.write_error(status_code, **kwargs)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 739, in write_error
        html = self.render_template("%s.html" % status_code, **ns)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 665, in render_template
        ns.update(self.template_namespace)
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 675, in template_namespace
        logged_in=self.logged_in,
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 198, in logged_in
        user = self.current_user
      File "/opt/conda/lib/python3.10/site-packages/tornado/web.py", line 1424, in current_user
        self._current_user = self.get_current_user()
      File "/opt/conda/lib/python3.10/site-packages/jupyter_server/base/handlers.py", line 177, in get_current_user
        raise RuntimeError(msg)
    RuntimeError: Calling `Template404.get_current_user()` directly is deprecated in jupyter-server 2.0. Use `self.current_user` instead (works in all versions).
[W 2024-12-14 09:22:35.150 ServerApp] 405 CONNECT api.ip.pn:443 (@185.91.127.81) 2.37ms referer=None
[I 2024-12-14 10:11:24.279 ServerApp] 302 GET / (@167.94.138.48) 0.84ms
[I 2024-12-14 10:11:27.281 ServerApp] Malformed HTTP message from 167.94.138.48: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 10:11:52.008 LabApp] 302 GET /lab (@167.94.138.48) 0.96ms
[I 2024-12-14 10:11:57.209 ServerApp] Malformed HTTP message from 167.94.138.48: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 10:11:59.965 ServerApp] 302 GET / (@167.94.138.48) 0.73ms
[I 2024-12-14 10:12:26.810 ServerApp] Malformed HTTP message from 167.94.138.48: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 10:12:31.534 ServerApp] 302 GET / (@167.94.138.48) 0.68ms
[I 2024-12-14 13:58:39.553 ServerApp] 302 GET / (@167.94.146.59) 0.80ms
[I 2024-12-14 13:58:49.183 ServerApp] 302 GET / (@167.94.146.59) 0.70ms
[I 2024-12-14 13:58:49.614 ServerApp] Malformed HTTP message from 167.94.146.59: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 13:58:52.861 LabApp] 302 GET /lab (@167.94.146.59) 0.96ms
[I 2024-12-14 13:58:52.873 ServerApp] Malformed HTTP message from 167.94.146.59: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 13:58:52.921 ServerApp] 302 GET / (@167.94.146.59) 0.67ms
[I 2024-12-14 13:58:58.850 ServerApp] Malformed HTTP message from 167.94.146.59: Malformed HTTP version in HTTP Request-Line: 'HTTP/2.0'
[I 2024-12-14 13:58:59.381 ServerApp] 302 GET / (@167.94.146.59) 0.70ms
slurmstepd: error: *** JOB 24828 ON worker5 CANCELLED AT 2024-12-14T18:14:09 DUE TO TIME LIMIT ***
